### ISSUE TITLE
[Enhancement] .NET Standard support

### ISSUE THREAD

**Issue by andreas-nesheim** [dannycabrera/Get-iOS-Model#16]
Is there any way this library could support .NET Standard? Not sure if it's possible, but it would be great to be able to use this from a class library.

Love the work, by the way!

**Comment by dannycabrera** [comment]
Hi @andreas-nesheim you have any Xamarion.iOS apps running .NET Standard?

**Comment by andreas-nesheim** [comment]
Not exactly. I have a Xamarin.iOS app and a Xamarin.Android app (native) which are using a shared .NET Standard library. To improve code reusage I would like to use your library from this .NET Standard project so I can toggle between using your library to get the correct iOS model if it's an iOS device and the same for Android.

**Comment by dannycabrera** [comment]
Gotcha, will take a deeper look as soon as I get some free time.

**Comment by peterfoot** [comment]
Another good use for this would be when retrieving the device model over Bluetooth from the Gatt DeviceInformation service ModelNumberString characteristic as this returns the same format and it would be great to convert this into something nicer for display.
